### PR TITLE
addSubstract Component

### PR DESCRIPTION
**Pull request by solymdev**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ x ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ x ] Storybook

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ x ] I have performed a self-review of my own code
- [ x ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "workbench.colorCustomizations": {
```
Comment (miguelromeroglb) [inline .vscode/settings.json]: Se necesita este archivo? creo que es alguna configuración propia de tu vscode, podrías excluirla del commit?

File: src/components/AddSubstract/addSubstract.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
+.Button {
```
Comment (miguelromeroglb) [inline src/components/AddSubstract/addSubstract.css]: creo que estos estilos ya existen para el component botón, no se estarían sobreescribiendo?

File: src/components/AddSubstract/AddSubstract.jsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            onClick={() => handleSubstract()}
+          >
+            <p className="control">-</p>
+          </button>
+        </div>
+        <div className="col-sm">
+          <p>{counter}</p>
+        </div>
+        <div className="col-sm">
+          <button
```
Comment (miguelromeroglb) [inline src/components/AddSubstract/AddSubstract.jsx]: Porque no usar elcomponent button que ya existe con los colores customizados?

**Review by miguelromeroglb — CHANGES_REQUESTED** [review]
Faltan los unit tests y hay unos comentarios que arreglar

File: .vscode/settings.json (hunk 1 above)
Comment (solymdev) [reply .vscode/settings.json]: Claro, excluyo la carpeta en el próximo pull request

File: src/components/AddSubstract/addSubstract.css (hunk 2 above)
Comment (solymdev) [reply src/components/AddSubstract/addSubstract.css]: Corregido

File: src/components/AddSubstract/AddSubstract.jsx (hunk 3 above)
Comment (solymdev) [reply src/components/AddSubstract/AddSubstract.jsx]: Listo

**Review by miguelromeroglb — APPROVED** [review]
LGTM

**Review by miguelromeroglb — APPROVED** [review]

**Pull request merged**